utils/index.sh

```diff
@@ -103,7 +103,7 @@ get_encryption_config(){
     # Check the apiserver for the encryption config
     # If encryption was never turned on, you won't find this config on the apiserver
     encrypted=false
-    encryption=$(oc get apiserver -o=json | jq -r '.items[0].spec.encrytion.type')
+    encryption=$(oc get apiserver -o=jsonpath='{.items[0].spec.encryption.type}' )
```

Comment:
Maybe you can get the object directly to simplify the expression:

```suggestion
    encryption=$(oc get apiserver cluster -o=jsonpath='{.spec.encryption.type}' )
```

Reply:
lol let me fire up another clusterbot to check